runtime/hsa-runtime/core/driver/kfd/amd_kfd_driver.cpp

```diff
@@ -289,5 +369,33 @@ void KfdDriver::MakeKfdMemoryUnresident(const void *mem) {
   hsaKmtUnmapMemoryToGPU(const_cast<void *>(mem));
 }
 
+bool KfdDriver::BindXnackMode() {
+  // Get users' preference for Xnack mode of ROCm platform.
+  HSAint32 mode = core::Runtime::runtime_singleton_->flag().xnack();
+  bool config_xnack = (core::Runtime::runtime_singleton_->flag().xnack() !=
```

Comment:
```suggestion
  bool config_xnack = (mode !=
```
or do you need to call it twice?

Reply:
Fixed